erc/ERCS/erc-7562.md

```diff
@@ -230,63 +330,72 @@ This means that a `Paymaster` and `Factory` contracts cannot practically be an "
 
 ### Unstaked Entities Reputation Rules
 
 * Definitions:
     * **`opsSeen`, `opsIncluded`, and reputation calculation** are defined above.
-    * `UnstakedReputation` of an entity is a maximum number of entries using this entity allowed in the mempool.
-    * `opsAllowed` is a reputation based calculation for an unstaked entity, representing how many `UserOperations` it is allowed to have in the mempool.
-* **[UREP-010]** `UserOperation` with unstaked sender are only allowed up to `SAME_SENDER_MEMPOOL_COUNT` times in the mempool.
+    * `UnstakedReputation` of an entity determines the maximum number of entries using this entity allowed in the mempool.
+    * `opsAllowed` is a reputation-based calculation for an unstaked entity, representing how many `UserOperations` it is allowed to have in the mempool.
+* **[UREP-010]** An unstaked sender is only allowed to have `SAME_SENDER_MEMPOOL_COUNT` `UserOperation`s  in the mempool. A staked sender is only limited by the SREP rules.
 * **[UREP-020]** For other entities: \
-    `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + (inclusionRate * INCLUSION_RATE_FACTOR) + (min(opsIncluded, 10000)`.
-    * This is a default of `SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT` for new entity
+  `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + (inclusionRate * INCLUSION_RATE_FACTOR) + (min(opsIncluded, 10000)`.
+    * This is a default of `SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT` for a new entity
 * **[UREP-030]** If an unstaked entity causes an invalidation of a bundle, its `opsSeen` is set to `1000`, effectively blocking it from inclusion for 24 hours.
 
 ### Alt-mempools rules:
 
-Alternate mempool is an agreed-upon rule that the bundlers may opt in to.
+Alternate mempool is an agreed-upon rule that the bundlers may opt into, in addition to the canonical mempool
+The alt-mempool "topic" is a unique identifier. By convention, this is the IPFS hash of the document describing (in clear test and YAML file) the specifics of this alt mempool
 
 * **[ALT-010]** The bundler listens to the alt-mempool "topic" over the P2P protocol
 * **[ALT-020]** The alt mempool rules MUST be checked only when a canonical rule is violated
     * That is, if validation follows the canonical rules above, it is not considered part of an alt-mempool.
+*  **[ALT-021]** Such a `UserOperation` (that violates the cannonical rules) is checked against all the "alternate mempools", and is considered part of all those alt-mempools 
 * **[ALT-030]** Bundlers SHOULD forward `UserOperations` to other bundlers only once, regardless of how many (shared) alt-mempools they have. \
-The receiving bundler validates the `UserOperations`, and based on the above rules (and subscribed alt-mempools) decides for which alt-mempools to "tag" it.
+  The receiving bundler validates the `UserOperations`, and based on the above rules (and subscribed alt-mempools) decides which alt-mempools to propagate it to.
 
+### Alt-mempool reputation:
+
+
```

Comment:
```suggestion
Alt-mempools are served by the same bundlers participating in the canonical mempool, but change the rules and may introduce denial-of-service attack vectors. To prevent them from taking the canonical mempool or other alt mempools down with them, a reputation is managed for each. An alt mempool that causes too many invalidations gets throttled. This limits the scope of the attack and lets the bundler continue doing its work for other mempools.
```

Reply:
@drortirosh I see you didn't add this suggestion to the latest version.  Do you think it's unneeded? I thought it would make sense to explain the rationale for this additional reputation.